### ISSUE TITLE
Parameters for ShanghaiTech Part A

### ISSUE THREAD

**Issue by srinadhu** [ZhihengCV/Bayesian-Crowd-Counting#15]
Hello @ZhihengCV ,

I changed the --crop-size to 256 as mentioned in the paper. While data-preparation the min_size is fixed as 256 and max_size as 5096. All the other parameters as default. I ran it on ShanghaiTech Part A. Unlike reported in paper I could only get an MAE of 66.0, that's a major difference of 3.2 I believe that the parameters of Sigma and background ratio might be of different values from default. Can you please let me know the changes from this codebase to that of used for ShanghaiTech Part A? Thanks.

**Comment by srinadhu** [comment]
hello @ZhihengCV ,  response would be greatly appreciated, it's been close to a month??

**Comment by zhangyuwei1996** [comment]
I also encountered the same problem. The experimental results on ShanghaiTech Part A are quite different from the results in the paper. I didn't get a result of mse <65 after running several times.

**Comment by ZhihengCV** [comment]
I upload the trained parameters and the new data loader for shanghaitech dataset. To get better results on shanghaitech A & B, you should use the new data loader, and for shanghaitech a, you should change learning rate to 1e-6, and bg_ratio to 0.1.

**Comment by ImMrMa** [comment]
Hello, how do you preprocess the Shanghai tech dataset so the dataloader can read? @zhangyuwei1996

**Comment by BoltenWang-Meta** [comment]
I set the parameters (lr=1e-6 and bg_ratio=0.1) for part A. The finally best MAE is 90.+. When setting the default parameters, the MAE is about 68. Is there some tricks else?

**Comment by xcaizewu** [comment]
Final Test: mae 349.8921880198049, mse 521.3918286187451
I get the results on test data, And when i use author best model , i get Final Test: mae 353.66828402844106, mse 534.9097216225637, why the experimental results on ShanghaiTech Part A are quite different from the results in the paper? @ZhihengCV @BoltenWang-Meta @zhangyuwei1996
